chef/cookbooks/ceph/libraries/default.rb

```diff
@@ -93,7 +93,9 @@ def get_mon_addresses()
 
     mons += get_mon_nodes()
     if is_crowbar?
-      mon_ips = mons.map { |node| Chef::Recipe::Barclamp::Inventory.get_network_by_type(node, "admin").address }
+      mon_ips = mons.map do |node|
+        Chef::Recipe::Barclamp::Inventory.get_network_by_type(node, "ceph_client").address
```

Comment:
I am not entirely sure about the purpose  of that network, but wouldn't it make sense to be able to reference any of the networks that are defined in the network barclamp here. By make the name an attribute on the ceph proposal? Instead of just hardcoding to "ceph_client"?

That way the user to choose whatever fits their enviroment best.